core/templates/pages/blog-admin-page/services/blog-admin-data.service.ts

```diff
@@ -25,11 +25,12 @@ import { BlogAdminPageData, BlogAdminBackendApiService } from 'domain/blog-admin
   providedIn: 'root'
 })
 export class BlogAdminDataService {
-  dataPromise: Promise<BlogAdminPageData>;
+  // 'dataPromise' is initialized before it is used in the class.
+  // We need to do non-null assertion.
```

Comment:
I am not sure what this  comment meant. Can you please elaborate ?

Reply:
updated